test/FastTests/Client/Queries/TimeSeries/BasicTimeSeriesQueries.cs

```diff
@@ -30,7 +30,7 @@ private class TsMapReduceIndexResult : TsMapIndexResult
             public long Count { get; set; }
         }
 
-        [RavenTheory(RavenTestCategory.Indexes | RavenTestCategory.TimeSeries)]
+        [RavenTheory(RavenTestCategory.TimeSeries)]
```

Comment:
those were categorized correctly

Reply:
I need then a good definition of what "indexes" is. The rules may not be so clear to me at least.